cypress/tests/ImageElement.spec.ts

```diff
@@ -42,6 +42,24 @@ describe("ImageElement", () => {
           expect(to).to.equal(5);
         });
       });
+      it("should update the document selection when the user focuses on the field and it contains content, even if the inner field's selection has not changed", () => {
+        addImageElement();
+        getDocSelection().then(([from]) => {
+          // The selection should be just after the inserted element
+          expect(from).to.be.greaterThan(13);
+        });
+        typeIntoElementField("caption", "Example");
+        focusElementField("caption");
+        focusElementField("src");
```

Comment:
`nit` we could make this test more robust by asserting that clicking the field, updates the selection. Otherwise, we can't be sure the selection has changed between at all been editing text and focusing fields.